packages/hoppscotch-backend/prisma/migrations/20251016080714_mock_server/migration.sql

```diff
@@ -0,0 +1,142 @@
+-- CreateEnum
+CREATE TYPE "WorkspaceType" AS ENUM ('USER', 'TEAM');
+
+-- CreateEnum
+CREATE TYPE "MockServerAction" AS ENUM ('CREATED', 'DELETED', 'ACTIVATED', 'DEACTIVATED');
+
+-- CreateTable
+CREATE TABLE "MockServer" (
+    "id" TEXT NOT NULL,
+    "name" TEXT NOT NULL,
+    "subdomain" TEXT NOT NULL,
+    "creatorUid" TEXT,
+    "collectionID" TEXT NOT NULL,
+    "workspaceType" "WorkspaceType" NOT NULL,
+    "workspaceID" TEXT NOT NULL,
+    "delayInMs" INTEGER NOT NULL DEFAULT 0,
+    "isPublic" BOOLEAN NOT NULL DEFAULT true,
+    "isActive" BOOLEAN NOT NULL DEFAULT true,
+    "hitCount" INTEGER NOT NULL DEFAULT 0,
+    "lastHitAt" TIMESTAMPTZ(3),
+    "createdOn" TIMESTAMPTZ(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updatedOn" TIMESTAMPTZ(3) NOT NULL,
+    "deletedAt" TIMESTAMPTZ(3),
+
+    CONSTRAINT "MockServer_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "MockServerLog" (
```

Comment:
Retention policies to limit logs from growing indefinitely via cron job or Postgres partitioning?